SpatialGDK/Documentation/content/tutorials/multiserver-shooter/tutorial-multiserver-cloudtest.md

```diff
@@ -14,16 +14,16 @@ An assembly is what’s created when you run `BuildWorker.bat`. They’re `.zip`
 **Note:** Unreal GDK projects default to using [Spatial for networking](). However, if the `bSpatialNetworking` option is present in your `DefaultGame.ini` configuration file (located in `<ProjectRoot>\Game\Config` directory), ensure that it is set to `True` (as in, `bSpatialNetworking=True`) to enable networking with Spatial for your cloud deployment.
 
 1. In a terminal window, change directory to the root directory of the Third-Person Shooter repository.
-1. Build a server-worker assembly by running: `Game\Plugins\UnrealGDK\SpatialGDK\Build\Scripts\BuildWorker.bat ThirdPersonShooterServer Linux Development ThirdPersonShooter.uproject`
-1. Build a client-worker assembly by running: `Game\Plugins\UnrealGDK\SpatialGDK\Build\Scripts\BuildWorker.bat ThirdPersonShooter Win64 Development ThirdPersonShooter.uproject`
+1. Build a server-worker assembly by running: `Game\Plugins\UnrealGDK\SpatialGDK\Build\Scripts\BuildWorker.bat GDKShooterServer Linux Development GDKShooter.uproject`
```

Comment:
We can ask them to use `BuildProject.bat` (gotta make sure it's updated first though)